pkgs/applications/misc/barrier/default.nix

```diff
@@ -5,13 +5,15 @@
 
 mkDerivation rec {
   pname = "barrier";
-  version = "2.4.0";
+  version = "2.4.0-4b12265ae5d";
```

Comment:
This should be `unstable-YYYY-MM-DD` for unstable versions.

I would, however prefer if we continued using 2.4.0 and just applied the fix as a patch by `fetchpatch`ing <https://github.com/debauchee/barrier/commit/4b12265ae5d324b942698a3177e1d8b1749414d7.patch> (which applies fine).

Reply:
Makes sense. I've made the changes. Thanks for reviewing!